tern/analyze/default/container/image.py

```diff
@@ -23,13 +23,14 @@
 logger = logging.getLogger(constants.logger_name)
 
 
-def load_full_image(image_tag_string):
-    '''Create image object from image name and tag and return the object'''
+def load_full_image(image_tag_string, load_until_layer=0):
```

Comment:
I think this could be changed to load_docker_image(...) as it doesnt necessarily load full image now

Reply:
discussed over slack, in context of this file it will be full image anyway thus leaving as-is